padlockcloud/whitelist.go

```diff
@@ -0,0 +1,42 @@
+package padlockcloud
+
+import (
+	"sort"
+	"sync"
+	"time"
+)
+
+// Server configuration
+type WhitelistConfig struct {
+	// Whitelist is on
+	On bool `yaml:"on"`
+	// Whitelisted path
+	Path string `yaml:"path"`
+	// Time duration to wait between updating whitelist
+	JobUpdateTime time.Duration `yaml:"job_update_time"`
+}
+
+type Whitelist struct {
+	// Whitelisted emails
+	Emails []string
```

Comment:
Why not use a `map[string]bool` for this?

Reply:
I have used a lot of maps in some of my other projects and not had any practice with the sort package or arrays in general, so I wanted to try something new. But yes I can change it over to map for this for that O(1), but O(log(n)) is not to bad either :+1: